src/healthcareapis/azext_healthcareapis/generated/_params.py

```diff
@@ -88,51 +107,51 @@ def load_arguments(self, _):
     with self.argument_context('healthcareapis private-endpoint-connection show') as c:
         c.argument('resource_group_name', resource_group_name_type)
         c.argument('resource_name', type=str, help='The name of the service instance.', id_part='name')
-        c.argument('private_endpoint_connection_name', options_list=['--name', '-n', '--private-endpoint-connection-nam'
-                                                                     'e'], type=str, help='The name of the private '
-                   'endpoint connection associated with the Azure resource', id_part='child_name_1')
+        c.argument('private_endpoint_connection_name',
+                   options_list=['--name', '-n', '--private-endpoint-connection-name'],
+                   type=str, help='The name of the private endpoint connection associated with the Azure resource',
+                   id_part='child_name_1')
 
     with self.argument_context('healthcareapis private-endpoint-connection create') as c:
         c.argument('resource_group_name', resource_group_name_type)
         c.argument('resource_name', type=str, help='The name of the service instance.')
-        c.argument('private_endpoint_connection_name', options_list=['--name', '-n', '--private-endpoint-connection-nam'
-                                                                     'e'], type=str, help='The name of the private '
-                   'endpoint connection associated with the Azure resource')
-        c.argument('private_link_service_connection_state_status', arg_type=get_enum_type(['Pending', 'Approved', ''
-                                                                                           'Rejected']), help=''
-                   'Indicates whether the connection has been Approved/Rejected/Removed by the owner of the service.')
-        c.argument('private_link_service_connection_state_description', type=str, help='The reason for '
-                   'approval/rejection of the connection.')
-        c.argument('private_link_service_connection_state_actions_required', type=str, help='A message indicating if '
```

Comment:
Can we keep `--private-link-service-connection-state-actions-required` for sometime so that customers can have time to update their code?